books/free-programming-books-ar.md

```diff
@@ -36,6 +36,7 @@
 ### C Sharp
 
 * [سبيلك المختصر لتعلم لغة c# - برمجة الواجهات](https://www.mobarmijoun.com/2014/04/c_19.html) - Khaled Al-Saadani (Arab Coders Academy)
+* Fundamentals-of-Computer-Programming-with-CSharp - ( https://www.introprogramming.info/wp-content/uploads/2013/07/Books/CSharpEn/Fundamentals-of-Computer-Programming-with-CSharp-Nakov-eBook-v2013.pdf) (pdf)- (author Svetlin Nakov)
```

Comment:
I could help you with resources format :wink:. Use this snippet

```markdown
* [FUNDAMENTALS OF COMPUTER PROGRAMMING WITH C# (The Bulgarian C# Programming Book)](https://www.introprogramming.info/wp-content/uploads/2013/07/Books/CSharpEn/Fundamentals-of-Computer-Programming-with-CSharp-Nakov-eBook-v2013.pdf) - Svetlin Nakov, Veselin Kolev et al. (PDF)
```

:information_source: Main authoritative site https://introprogramming.info/english-intro-csharp-book/ which this PDF can be found in.


:warning: But the **most important**. this resource is written in English. **Move it right listing**: [`free-programming-books-langs.md#c-sharp`](https://github.com/Debajyotighosh654/free-programming-books/blob/patch-2/books/free-programming-books-langs.md#c-sharp)